enos/enos.py

```diff
@@ -117,6 +119,10 @@ def up(env=None, **kwargs):
     base_inventory = seekpath(env['config']['inventory'])
     inventory = os.path.join(env['resultdir'], 'multinode')
     generate_inventory(env['rsc'], base_inventory, inventory)
+
+    if kwargs['--skip-install']:
```

Comment:
Thanks for the clarification :)

I think the right direction here is to addres #142. For the time being Enos isn't designed for anything else than deploying OpenStack. As a side effect of solving #142 you'll be able to use the `up` phase for what you want, and you'll get and `env` properly populated et thus reusable for anything else. 
A further step would be to extract the reservation part from EnOS could be interesting in itself but is not our main focus now so that this logic could be used in another context.

I'm closing this : don't hesitate to come on https://gitter.im/BeyondTheClouds/enos if you are interested in the previous items.